### ISSUE TITLE
In the coash report is it possible to display the question too? - Version 0.15.1 - Feature request.

### ISSUE THREAD

**Issue by harishbalachandran** [learningequality/ka-lite#5044]
This is not a bug, but a feature request.

**Detailed Description:**
In the coach report when the teacher wants more information about how a student has done in a particular exercise, he/she clicks on the progress to see the response given for each question. 

**Can we have questions also displayed instead of only the answers?**

This will help the teachers to understand which question did the student go wrong and it is a key feature because there is no point displaying only the answers when the teacher does not know the respective question. 

![screen shot 2016-03-25 at 11 29 14 am](https://cloud.githubusercontent.com/assets/13453618/14038930/d857cefa-f27c-11e5-9a55-26d75365b3cb.png)
![screen shot 2016-03-25 at 11 29 47 am](https://cloud.githubusercontent.com/assets/13453618/14038939/ecb032a2-f27c-11e5-9b96-ca73acb0c665.png)

**Comment by rtibbles** [comment]
Yes, I can see the use case for this, certainly. Clicking on the Exercise link does not take you to the particular question in question.
